navigation/MainTabNavigator.js

```diff
@@ -102,10 +107,28 @@ ProfileStack.navigationOptions = {
 
 ProfileStack.path = "";
 
+const ImageStack = createStackNavigator(
```

Comment:
Is this here just for testing?

If so, let's remove it before the merge, that way we don't have extra tabs on the `MaintabNavigator`